scripts/build-probe-binaries

```diff
@@ -462,6 +478,17 @@ function debian_build {
 	cd ${BASEDIR}
 }
 
+# Check out the required tag of each branch. Only needed once.
+checkout_code_for draios/sysdig
+
+if [ $PROBE_NAME != "sysdig-probe" ]; then
+    checkout_code_for falcosecurity/falco
```

Comment:
This will be a problem when we call `update_code_for` because it expects to find the agent tags, so we would need to use `draios/oss-falco`

I would like to build probes directly from `falcosecurity/falco`, but that would make for more special casing in this script. If we can make sure that `draios/oss-falco` is always in sync, we can keep using it throughout the script.

Reply:
brett and I chatted in person and I'm going to try to narrow down this script to *only* use the sysdig repo. The only think we need the other repos for is to specify the probe name, which seems like a lot of overkill.